css/bootstrap/_navbar.scss

```diff
@@ -64,6 +61,7 @@
   padding-left: 0;
   margin-bottom: 0;
   list-style: none;
+  background-color: white;
```

Comment:
I'm pretty sure this file is part of bootstrap, so can we do any custom stuff in the (admittedly poorly named) `bootstrap.scss` file? I think that's where all the other custom stuff lives.